Tests/LibWeb/Ref/input/wpt-import/css/css-contain/contain-paint-stacking-context-001a.html

```diff
@@ -1,66 +0,0 @@
-<!DOCTYPE HTML>
```

Comment:
Why are you deleting a test here? Parts of the PR description should probably end up in the commit messages so they become self-explanatory :-)

Reply:
Note that instead of deleting a test, you can also mark it as skipped and link a new GitHub issue to it.

Reply:
I've added the explanation to the commit description now, but I'll still remove the test for now since I want to fix it in the WPT repo, then re-import it.
The failure also happens in Chromium if I apply SerenitySans to the test case so it's really a flaky WPT test and not something we can fix. (beyond not using SerenitySans in the test runner, I guess)